discovery/consul/consul.go

```diff
@@ -517,6 +519,12 @@ func (srv *consulService) watch(ctx context.Context, ch chan<- []*targetgroup.Gr
 			labels[serviceMetaDataLabel+model.LabelName(name)] = model.LabelValue(v)
 		}
 
+		// Add all key/value pairs from the service's tagged addresses as their own labels
```

Comment:
Comments should end in a full stop.

Reply:
There you go; fixed a few comments more in the process. Thanks!